composer.json

```diff
@@ -1,6 +1,5 @@
 {
     "name": "phile-cms/phile",
-    "version": "1.6.0",
```

Comment:
> So the trick is, to tag the version in github

I expected the same and removed the line in 1.5.0. Afaik it was tagged properly on github/master, but I couldn't make it work and reverted in 1.5.2.

Maybe I missed something, but a little heads up that there may be dragons. :blush: